src/ray/core_worker/core_worker.h

```diff
@@ -186,6 +187,7 @@ struct CoreWorkerOptions {
   int runtime_env_hash;
   /// The PID of the process for setup worker runtime env.
   pid_t worker_shim_pid;
+  StartupToken startup_token;
```

Comment:
Please add a comment above describing what this does :)

Reply:
I will add documentation comments at all the necessary places once I complete testing. There are some internal API changes so tests are required to be updated. Thanks.